### ISSUE TITLE
Odyssey Stats: duplicated params added to hash on refreshing of the page

### ISSUE THREAD

**Issue by kangzj** [Automattic/wp-calypso#71171]
This is a `page.js` bug and has low priority as it doesn't affect any functionality.

![Image](https://user-images.githubusercontent.com/1425433/207500081-689f396d-e2c1-42f7-a2e9-eb61caff552d.png)

**Comment by kangzj** [comment]
I don't see a clear way to fix the bug without touching `page.js`. I wonder if you folks have any suggestions. @Automattic/red

**Comment by jsnmoon** [comment]
@kangzj, what do you think about dropping this line into [`app.jsx:50`](https://github.com/Automattic/wp-calypso/blob/4bfad67a35e90202aaf5c51a28641b1d2008031a/apps/odyssey-stats/src/app.jsx#L49)?

```js
window.location.hash = window.location.hash.replaceAll( '?page=stats', '' );
```

It doesn't solve the issue of `?page=stats` being appended to the hash, but we can at least clear it on new page loads.

**Comment by kangzj** [comment]
@jsnmoon ha this actually works well. That's super helpful. Thanks!

**Comment by kangzj** [comment]
closed by #74018
